src/bugsnag-link.js

```diff
@@ -97,10 +98,11 @@ class BugsnagLink extends stream.Subscriber {
         }
         // if error found
         if (error) {
+          error = cloneError(error);
```

Comment:
why overwrite the same Error  object with it's clone?

Reply:
To not modify the original error by link, but do that to a clone. This led to an issue when the original error object's message was modified with prefix